integration_tests/src/main/python/regexp_test.py

```diff
@@ -808,7 +808,7 @@ def test_regexp_extract_all_idx_negative():
 
 @allow_non_gpu('ProjectExec', 'RegExpExtractAll')
 def test_regexp_extract_all_idx_out_of_bounds():
-    gen = mk_str_gen('[abcd]{0,3}')
+    gen = mk_str_gen('[a-d]{1,2}.{0,1}[0-9]{1,2}')
```

Comment:
Generally the change looks good. I just want to check that you changed the generation pattern to make it so it would always match regular expression below.